app/src/main/java/com/example/parkingsystemjava/mvp/model/ParkingReservationModel.java

```diff
@@ -0,0 +1,7 @@
+package com.example.parkingsystemjava.mvp.model;
+
+import com.example.parkingsystemjava.mvp.contract.ParkingReservationContract;
+
+public class ParkingReservationModel implements ParkingReservationContract.ReservationActivityModel {
```

Comment:
if you are not using this now, it should not be here. Please remove it

Reply:
done